packages/rocketchat-api/server/v1/misc.js

```diff
@@ -175,3 +175,38 @@ RocketChat.API.v1.addRoute('directory', { authRequired: true }, {
 		});
 	},
 });
+
+RocketChat.API.v1.addRoute('invite.email', { authRequired: true }, {
+	post() {
+		if (!this.bodyParams.email) {
+			throw new Meteor.Error('error-email-param-not-provided', 'The required "email" param is required.');
+		}
+
+		Meteor.runAsUser(this.userId, () => Meteor.call('sendInvitationEmail', [this.bodyParams.email]));
+		return RocketChat.API.v1.success();
+
+		// sendInvitationEmail always returns an empty list
```

Comment:
Is this debug code you have commented out?  Do we need it, or can you remove?

Reply:
The already existing sendInvitationEmail method should ideally return a list of email addresses to which email was successfully sent, but it is currently returning an empty list. The code commented out is to send accurate response by the API, whether the Email has been sent or not, but fails to do so due to the sendInvitationEmail method returning empty list every time. So, once the sendInvitationEmail method is fixed, we can uncomment the code.

Also, there is a possibility that it sendInvitationEmail method works fine when the SMTP is configured, I just tested it without it.

Reply:
Are we tracking this issue somewhere?  Is it something we should fix?  Perhaps we should make a story for it on our contacts project board?

Reply:
We currently are not tracking it. I'll once again reproduce and confirm it and then will create an issue upstream, and then refer it to our board.

Reply:
Awesome!  Thanks.

Reply:
https://github.com/RocketChat/Rocket.Chat/issues/12923